doc/try_new_features_before_release.md

```diff
@@ -29,4 +29,25 @@ Step 1. Open a PR no matter it is merged or not. Click "Checks" tab.
 
 ![](assets/5.PNG)
 
-Step 2-4. Follow the same steps of tutorial for commit.
\ No newline at end of file
+Step 2-4. Follow the same steps of tutorial for commit.
+
+## When Azure Python SDK is not released
+
+When a feature is in private preview stage, the corresponding Azure Python SDK cannot be released. But you can still try the new feature.
+
+Click pypi and download all `.whl` files in artifact page. Install it along with the Azure Python SDK. The SDK can be retrieved from Azure SDK team. You can request a release of SDK in https://portal.azure-devex-tools.com/. Since it's in private preview, it cannot be released. You can get a private build instead.
+```
+pip install azure_cli-2.6.0-py3-none-any.whl azure_cli_command_modules_nspkg-2.0.3-py3-none-any.whl azure_cli_core-2.6.0-py3-none-any.whl azure_cli_nspkg-3.0.4-py3-none-any.whl azure_cli_telemetry-1.0.4-py3-none-any.whl azure_mgmt_imagebuilder-1.0.0-py2.py3-none-any.whl
+```
+`azure_cli-2.6.0-py3-none-any.whl`, `azure_cli_command_modules_nspkg-2.0.3-py3-none-any.whl`, `azure_cli_core-2.6.0-py3-none-any.whl`, `azure_cli_nspkg-3.0.4-py3-none-any.whl`, `azure_cli_telemetry-1.0.4-py3-none-any.whl` are packages of Azure CLI. `azure_mgmt_imagebuilder-1.0.0-py2.py3-none-any.whl` is a package of Image Builder resource provider. You should change it to your own SDK package containing the new feature.
+
+We recommend using a virtual environment to install the .whl files. This is an optional step.
+Example commands in PowerShell:
+```
+# Create a virtual environment 
+python -m venv env
+# Activate it
+env\Scripts\activate.ps1
+```
+
+Unfortunately, `.msi` and `.deb` files are not available since Azure Python SDK is not released and cannot be found.
```

Comment:
Actually you can. With msi, you can first install the msi from artifact page, then install SDK with the bundled python, for instance in Powershell:
`& 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe' -m pip install --upgrade azure_mgmt_imagebuilder-1.0.0-py2.py3-none-any.whl`

Not tested with deb, but similar command.

Reply:
Cool! Let me try it.

Reply:
I will update this part in another PR.